### PR TITLE
#151948282 Unresolved Issues Dashboard

### PR DESCRIPTION
**Pull request by Sekams**
#### What does this PR do?
Delivers the dashboard for unresolved issues feature
#### Description of Task to be completed?
The unresolved issues dashboard shows the user all the failed jobs of that user. It can be accessed at the url `unresolved/` or clicking the 'UNRESOLVED' tab
#### How should this be manually tested?
After cloning the repository and navigating to the location of the cloned project and activating the virtual environment, run:
```./manage.py test```
#### Any background context you want to provide?
This feature is independent of the sent notifications to the user about the failed jobs
#### What are the relevant pivotal tracker stories?
[#151948282](https://www.pivotaltracker.com/story/show/151948282)
#### Screenshots (if appropriate)
screen shot 2017-11-02 at 3 17 59 pm

#### Questions:

### CODE REVIEW

**Review by ivantoz — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -10,6 +10,8 @@ djmail==0.11.0
 futures==3.0.3
 gunicorn==19.7.1
 lxml==4.1.0
+mock==2.0.0
```
Comment (ivantoz) [inline requirements.txt]: Where are you using this?

File: requirements.txt [hunk 2]
```diff
@@ -10,6 +10,8 @@ djmail==0.11.0
...
+pbr==3.1.1
```
Comment (ivantoz) [inline requirements.txt]: What is this for?

**Review by ivantoz — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (sekams) [reply requirements.txt]: It is needed in several test files including:
- hc/api/tests/test_notify.py
- hc/api/tests/test_sendalerts.py
- hc/front/tests/test_add_pushbullet.py
- hc/front/tests/test_add_slack.py
- hc/payments/tests/test_billing.py
- hc/payments/tests/test_cancel_plan.py
- hc/payments/tests/test_create_plan.py
- hc/payments/tests/test_get_client_token.py
- hc/payments/tests/test_invoice.py

File: requirements.txt (hunk 2 above)
Comment (sekams) [reply requirements.txt]: pbr is required by the mock package and the usage of the mock package is described in the comment above.

**Pull request closed without merging**